src/controllers/index.js

```diff
@@ -56,6 +57,15 @@ router.get('/', function(req, res) {
         res.send(generatePage(HomePage, store));
```

Comment:
Let's modify generatePage so it takes in as a parameter what route it's generating. That way, we can change it so that instead of `<script src="js/bundle.js"></script>` it will do `<script src=" + route + /js/bundle.js"></script>`